packages/ios-client/Sources/FishjamClient/models/Endpoint.swift

```diff
@@ -0,0 +1,61 @@
+public enum EndpointType: String, Codable {
+    case WEBRTC
+
+    init(fromString s: String) {
+        switch s.lowercased() {
+        case "webrtc":
+            self = .WEBRTC
+        default:
+            self = .WEBRTC
+        }
```

Comment:
**suggestion:** Maybe something like this will be easier to maintain? No need to add future cases to the switch. 

`self = EndpointType(rawValue: s.uppercased()) ?? .WEBRTC` it

Reply:
On the one hand i agree with it, but on other hand adding new endpoint would require much more work and adding a new case at least in this enum. 
Although i like your solution so i wil use it.